### ISSUE TITLE
Can't Open Map

### ISSUE THREAD

**Issue by LifeHckr** [CyanideX/Infernal-Skies#53]
Whenever I try to open the map to play with my friend on LAN it returns me to the main menu no matter what. If I have to pay for a server thats bullshit because I'm not rich with enough money to waste on a server.

**Comment by dmillerw** [comment]
Modded Minecraft and LAN don't usually play well together

**Comment by LifeHckr** [comment]
No, I can't even get the opportunity to open LAN because when I open the world it automatically returns to the main menu.

**Comment by CyanideX** [comment]
Yeah, I had this issue a while back. You need to download the latest map from Curse.

**Comment by LifeHckr** [comment]
I'm 99% sure I have the latest map and it still won't work no matter what.
